rhel7/profiles/e8.profile

```diff
@@ -0,0 +1,132 @@
+documentation_complete: true
+
+title: 'Essential Eight System Hardening for Red Hat Enterprise Linux 7'
```

Comment:
@shaneboulden Since the Essential 8 is not a global standard (similar to ``ospp``/Common Criteria), should ``Australian`` be included in the profile title? We lack a formal naming scheme, but that is also how the DISA RHEL 7 STIG profile is named (``DISA RHEL 7 STIG`` vs ``RHEL 7 STIG``). 

E.g.:
- Australian Essential Eight System Hardening for Red Hat Enterprise Linux 7
- Australian Cyber Security Centre Essential Eight for Red Hat Enterprise Linux 7

Reply:
(same comment for RHEL 8 profile)

Reply:
Personally, I'd prefer not to have Australian in the title. Doesn't New Zealand also use Essential Eight?

Reply:
Thanks @shawndwells and @redhatrises for looking at this.

I've just rebased this PR with an updated title for each profile, including 'Australian Cyber Security Centre (ACSC)'.

@redhatrises looking at the other ComplianceAsCode profiles, they seem to be named after the organisation that owns/maintains the standard, rather than who it is used by. In this case, while NZ organisations [may use](https://www.sss.co.nz/news-and-updates/asds-essential-eight/) the Essential Eight, it's still owned/maintained by the ACSC. This looks to align with profiles like the 'DISA STIG'.